src/components.d.ts

```
@@ -1037,6 +1045,26 @@ declare global {
         prototype: HTMLRevogrFilterPanelElement;
         new (): HTMLRevogrFilterPanelElement;
     };
+    interface HTMLRevogrFilterPopupElementEventMap {
+        "close": void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)